images/linux/scripts/installers/action-archive-cache.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+################################################################################
+##  File:  action-archive-cache.sh
+##  Desc:  Download latest release from https://github.com/actions/action_verions
+##         and un-tar to /opt/actionarchivecache
+################################################################################
+
+# Prepare directory and env variable for ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE
+ACTION_ARCHIVE_CACHE_DIR=/opt/actionarchivecache
+mkdir -p $ACTION_ARCHIVE_CACHE_DIR
+echo "ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE=$ACTION_ARCHIVE_CACHE_DIR" | tee -a /etc/environment
```

Comment:
you can use the `addEtcEnvironmentVariable` helper defined here https://github.com/actions/runner-images/blob/main/images/linux/scripts/helpers/etc-environment.sh